modules/user-access/pages/user-management.adoc

```diff
@@ -268,6 +264,162 @@ GSQL > REVOKE ROLE role1, role2 ON GRAPH example_graph
 
 The above command will revoke roles `role1` and `role2` on graph `example_graph` from users `user1` and `user2`.
 
+== Grant privileges to a user
+
+Users with the `WRITE_USER` privileges on global can grant RBAC privileges to the users.
+
+=== Syntax
+
+[source,text]
+----
+GRANT <privilegeName1> (, <privilegeName2>)* ON <privilegeObjects>
+        [IN <privilegeScopes>]? TO <user_name1> (, <user_name2>)*
+----
+
+=== Require privilege
+
+`WRITE_USER`
+
+=== Example
+
+. To grant privileges to a user, run the `GRANT PRIVILEGE` command from the GSQL shell:
++
+[source,text]
+----
+GSQL > GRANT WRITE ON ALL ROLES
+        IN GRAPH example_graph TO user1 , user2
+----
+
+This will allow `user1` and `user2` to modify roles on the graph `example_graph`. To see a full list of privileges and the command they allow users to run, see 
```

Comment:
Be specific after this. It is not clear.

It should be like: This command allows user1 and user2 to modify roles on the example_graph. Specifically, they can perform operations that involve creating, modifying, or deleting roles within the graph.

Reply:
This is from our current docs. Whatever, I can improve it here.